firmware/main/conductor/conductor.ino

```diff
@@ -24,29 +75,221 @@ bool playbackRunning = true;
 typedef uint8_t staff_data_t;
 Staff <staff_data_t> staff;
 
+String instrument, note, sound;
+String fileType = ".WAV";
+char fileName[13];
+
+
 void onCartReception(cart_t* cart, void* args){
   staff[cart->col][cart->row] = cart->val;
 //  staff.showStaffDebug();
 }
 
+static volatile uint32_t playbackColumn = 0;
+//int bars = 0;
+
 void playColumn( void ){
-  static volatile uint32_t playbackColumn = 0;
+  //make player in here
+  random_cart();
+  //random_cart();
+  //random_cart();
+  
+  
+  staff[playbackColumn][cart.row] = cart.val;
   if(playbackRunning){
     staff_data_t* data = staff[playbackColumn];
-  
+
+    
+    playSdWav1.stop();
+    playSdWav2.stop();
+    playSdWav3.stop();
+    playSdWav4.stop();
+    playSdWav5.stop();
+    playSdWav6.stop();
+    playSdWav7.stop();
+
+    //the audio player would crash after 3 playthroughs, for some reason, adding these two lines prevents that
+    AudioNoInterrupts();
+    AudioInterrupts();
```

Comment:
We should investigate this further. It may be better to put ```AudioNoInterrupts``` at the beginning and ```AudioInterrupts``` at the end.